Doc/library/tkinter.rst

```diff
@@ -799,6 +799,10 @@ reference to the image. When the last Python reference to the image object is
 deleted, the image data is deleted as well, and Tk will display an empty box
 wherever the image was used.
 
+.. seealso::
+
+    The `Pillow package <https://pillow.readthedocs.io/>`_ adds support for
```

Comment:
The home page of Pillow is http://python-pillow.org/.

And I think that the link should be from just `Pillow`.

Reply:
Home page updated, I don't understand what you mean by "the link should be from just Pillow".

Reply:
Oh, you mean it should be rendered as: "The <a ...>Pillow</a> package"?

Reply:
@serhiy-storchaka the comment you made "the link should be from just Pillow" remains unaddressed. Could you please elaborate?

Reply:
I thought it would look better as
```rst
The `Pillow <https://pillow.readthedocs.io/>`_ package
```
But this is not strong preference.

Reply:
Yes, I agree. I'll change it.

Reply:
Done.